addons/web/static/src/views/basic_relational_model.js

```diff
@@ -505,10 +505,14 @@ export class Record extends DataPoint {
             data[fieldName] = mapWowlValueToLegacy(value, fieldType);
         }
         if (this._urgentSave) {
-            return this.model.__bm__.notifyChanges(this.__bm_handle__, data, {
+            const fieldNames = await this.model.__bm__.notifyChanges(this.__bm_handle__, data, {
                 viewType: this.__viewType,
                 notifyChange: false,
             });
+            resolveUpdatePromise();
+            this._removeInvalidFields(fieldNames);
+            this.__syncData();
+            return fieldNames;
```

Comment:
not necessary I think (`return;` should be enough)

Reply:
done